### PR TITLE
PLAST with foss2016a

### PR DESCRIPTION
**Pull request by RvDijk**

### CODE REVIEW

File: easybuild/easyconfigs/p/PLAST/PLAST-2.3.1-foss-2016a.eb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+easyblock = 'CMakeMake'
+
+name = 'PLAST'
+version = '2.3.1'
+
+homepage = "http://www.irisa.fr/symbiose/projects/plast/" 
+description = """
+PLAST is a parallel alignment search tool for comparing large protein banks
```
Comment (boegel) [inline easybuild/easyconfigs/p/PLAST/PLAST-2.3.1-foss-2016a.eb]: no need to wrap, use single line?

File: easybuild/easyconfigs/p/PLAST/PLAST-2.3.1-foss-2016a.eb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+"""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = ['https://github.com/PLAST-software/plast-library/releases/download/v2.3.1/']
```
Comment (boegel) [inline easybuild/easyconfigs/p/PLAST/PLAST-2.3.1-foss-2016a.eb]: use `%(version)s` rather than hardcoding the version please

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2496.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/55fca242b8114a7b7fcd1ac3492691db for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node2099.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/1d5f54c306c24b0bb29162dee9bba875 for a full test report.

**Comment by boegel** [comment]
@RvDijk missing dep?

```
CMake Error at /user/scratchdelcatty/gent/gvo000/gvo00002/vsc40023/easybuild_REGTEST/SL6/sandybridge/software/CMake/3.5.2-foss-2016a/share/cmake-3.5/Modules/FindPackageHandleStandardArgs.cmake:148 (message):
  Could NOT find JNI (missing: JAVA_AWT_LIBRARY JAVA_JVM_LIBRARY
  JAVA_INCLUDE_PATH JAVA_INCLUDE_PATH2 JAVA_AWT_INCLUDE_PATH)
```

**Comment by RvDijk** [comment]
@boegel Perl only is used in order to set the date, should it be in the name of this easyconfig? I guess that Perl is a build dependency, but %(perlver)s does not work in build dependencies.

**Comment by boegel** [comment]
@RvDijk what do you mean exactly with `set the date`? It does sound like Perl is just a build dep here, and in that case if should not be in the versionsuffix indeed.

I would consider `%(perlver)s` not working for build dependencies a bug though (although a low priority one), so please open an issue for that at https://github.com/hpcugent/easybuild-framework/issues

File: easybuild/easyconfigs/p/PLAST/PLAST-2.3.1-foss-2016a-Java-1.8.0_92.eb [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+source_urls = ['https://github.com/PLAST-software/plast-library/releases/download/v%(version)s/']
+sources=['%(namelower)s_source_v%(version)s.tar.gz']
+
+builddependencies = [
+    ('CMake', '3.5.2')
+]
+
+dependencies = [
+    ('Java', '1.8.0_92', '',  True),
+    ('Perl', '5.22.1')
```
Comment (boegel) [inline easybuild/easyconfigs/p/PLAST/PLAST-2.3.1-foss-2016a-Java-1.8.0_92.eb]: @RvDijk still a runtime dep (in disagreement with your commit message)?

**Comment by RvDijk** [comment]
@boegel it seems that Perl only is used to retrieve the date and put it in a variable. Therefore putting it in the build dependencies is fine.

File: easybuild/easyconfigs/p/PLAST/PLAST-2.3.1-foss-2016a-Java-1.8.0_92.eb (hunk 3 above)
Comment (boegel) [inline easybuild/easyconfigs/p/PLAST/PLAST-2.3.1-foss-2016a-Java-1.8.0_92.eb]: nvm, spoke too soon :)

File: easybuild/easyconfigs/p/PLAST/PLAST-2.3.1-foss-2016a-Java-1.8.0_92.eb [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+homepage = "http://www.irisa.fr/symbiose/projects/plast/" 
+description = """PLAST is a parallel alignment search tool for comparing large protein banks"""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = ['https://github.com/PLAST-software/plast-library/releases/download/v%(version)s/']
+sources=['%(namelower)s_source_v%(version)s.tar.gz']
+
+builddependencies = [
+    ('CMake', '3.5.2')
```
Comment (boegel) [inline easybuild/easyconfigs/p/PLAST/PLAST-2.3.1-foss-2016a-Java-1.8.0_92.eb]: @RvDijk missing `,` at the end of this line, leading to a parse error: `TypeError: 'tuple' object is not callable`

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2130.delcatty.os - Linux SL 6.8, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/a7fdeac54a5cba21da76e7a74b77c08e for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2539.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/067dd2f215ceee374ed4f1f8d6bda85f for a full test report.

**Comment by boegel** [comment]
Going in, thanks @RvDijk!

**Pull request merged**